### ISSUE TITLE
environment setup question

### ISSUE THREAD

**Issue by erikjthomas** [fastlane/fastlane#6559]
### New Issue Checklist
- [X] Updated fastlane to the latest version
- [X] I have read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)
### Issue Description

Is anyone using eclipse with Ruby DLTK plugin? Creating fastlane dev project, downloaded latest fastlane zip. The Ruby plugin is tagging 377 errors related to syntax errors on colons. Using ruby 2.3.1p112 on OSX 10.12. Greatly appreciate tips on setting up. I could use Sublime, but would like to use in eclipse if possible. Not setting up to contribute yet, just want to step code to explore/learn for possible future contribution.
##### Complete output when running fastlane, including the stack trace and command used

```
N/A - fastlane runs fine, just trying to set up dev env
```
### Configuration Files

Please copy the complete content of your `Fastfile` and any other configuration files you use below:

Fastfile:

``` ruby
# All default content in latest fastlane-master.
```
### Environment

fastlane version (run `fastlane -v`): fastlane 1.105.2

Do you use bundler to execute fastlane (i.e. `bundle exec fastlane`)? No

Do you use a Ruby environment manager (e.g. `chruby`, `rbenv`, `rvm`)? Yes: rbenv

What Xcode version do you use (e.g. 7.3, 8.0) 8.0

**Comment by TKBurner** [comment]
@erikjthomas I haven't tried this config, but we'll leave this open in case someone can help you.

**Comment by erikjthomas** [comment]
Thanks!

Erik Thomas
Chief Architect
Office: 541.247.2995 / Mobile: 303.304.1466



http://linqto.com <http://linqto.com/>

This email may contain confidential and privileged material for the sole use of the intended recipient. Any review or distribution by others is strictly prohibited. If you are not the intended recipient, please contact the sender and delete all copies.

**Comment by erikjthomas** [comment]
After experimenting with various ruby development tools and editors, I settled on RubyMine from JetBrains (makers if IntelliJ) and it's working quite well. I got what I wanted which was an integrated debugging (stepping code) setup which contributes to faster development velocity, at least in my case. I recommend it as a great IDE for ruby, especially for folks like me (15 years JEE web app development) who are used to using a full on IDE with debug configurations, project management, integrated git and svn, etc.

Closing this issue now.
